README.md

```diff
@@ -169,26 +165,6 @@ This will generate the binaries inside the `dist_mist/release` or `dist_wallet/r
 
 #### Options
 
-##### platform
```

Comment:
Why has this entire section been removed? You can actually still build for each platform or all together.